kuiper-game/Systems/CaptainsConsole.cs

```diff
@@ -81,15 +111,15 @@ private void Ship(string subChoice)
         private void SetCourse()
         {
             ConsoleWriter.Write($"What location should the ship set a course for?");
-            foreach (var location in Locations.Destinations)
+            foreach (var location in Locations.Destinations) //A change for github
```

Comment:
@Nicoolai any suggestions on how i can get a list of locations the ship can set a course towards? all bodies for example. Perhaps convinienced to `CelestialBodies Where Parent is Sun or CurrentPlanets bodies`

Reply:
Yes, I think we should just extend SolarSystemService. 
GetAllBodies() is of course super easy to make.
I already added some GetBodies.. methods but didn't implement them yet.
But I like @VisualBean's idea where `SolarSystemService` maps when you create and then knows what type to get.
Or maybe the selection filter, for `GetBodies(CelestialBodyType type)` should just be a typeof instead of the enum (maybe have both).